ansible/roles/openshift_firstboot_scripts/files/insights-register.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+while getopts c:t: option
+do
+case "${option}"
+in
+c) CLUSTERNAME=${OPTARG};;
+t) NODETYPE=${OPTARG};;
+esac
+done
+
+INTERNAL_HOSTNAME=$(curl 169.254.169.254/latest/meta-data/local-hostname)
+
+/usr/bin/redhat-access-insights --register --display-name=$CLUSTERNAME-$NODETYPE-$INTERNAL_HOSTNAME --group=$CLUSTERNAME
```

Comment:
should we add this to the current subscription manager script?  I don't have a feeling one way or another, just a thought...

2nd issue, should we background this?

Reply:
Good call. The timeout for this command is much lower than it used to be, but still isn't time we should spend waiting if we don't need to. I updated the command to background it.